src/components/SearchFilter.vue

```diff
@@ -76,11 +99,13 @@
 
           <QueryableInput
             v-for="(filter, index) in filters" :key="filter.id"
-            :value.sync="filter.value"
-            :operator.sync="filter.operator"
+            :value="filter.value"
+            :operator="filter.operator"
             :queryable="filter.queryable"
             :index="index"
             :cql="cql"
+            @update:value="val => filter.value = val"
+            @update:operator="op => filter.operator = op"
```

Comment:
Why not v-model:value and v-model:operator? It looks like that is the intended was of doing it?

Reply:
We need to check whether the linter actually works correctly in v3.